Source/utils/utf8.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <cstdint>
+#include <cstring>

```

Comment:
This is only for the `size_t` typedef? `<cstddef>` carries less baggage given most of these functions deal with `string` derivatives.

Reply:
`std::nemcpy`

Reply:
... but i moved that to the cpp file